app/assets/stylesheets/notification_item.scss

```diff
@@ -6,19 +6,12 @@
     @include has-fa-icon(bell, solid);
     @include has-fa-icon(circle, solid, after);
 
-    &::before {
```

Comment:
On small devices, the unread-notifications icon overlaps with the link text after removing the default margin. I think we have to recover the margin for this icon on small devices. 🤔  Something like this should work:
```
    &::before {
      @include breakpoint(small only) {
        margin-right: $font-icon-margin;
      }
    }
```


**Before**
<img width="419" alt="Notification-Icon-margin-error-small" src="https://user-images.githubusercontent.com/15726/123925124-e70c7680-d98a-11eb-9011-7993a328f19b.png">


**After**
<img width="419" alt="Notification-Icon-Fix" src="https://user-images.githubusercontent.com/15726/123925152-ec69c100-d98a-11eb-93ab-71ffbe31f8de.png">

Reply:
:+1: [Changes applied](https://github.com/consul/consul/compare/7e9d06e0..db7c1c344cf).